.github/workflows/go.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest]
-        go-version: [1.19, "1.20"]
+        go-version: ["1.20", "1.21"]
```

Comment:
Why are we doing this here?

Reply:
this was before i root caused my issue. and thought this might fix it. but i left it because 1.21 is the latest released version. all tests were passing with it, but i can change it back if you think we should

Reply:
Yeah revert this. We technically still need to check 1.19 and I'm going to handle this in a separate PR today anyway.